google-oauth-client/src/main/java/com/google/api/client/auth/oauth2/AuthorizationCodeFlow.java

```diff
@@ -560,21 +597,42 @@ public static class Builder {
      * @param clientId client identifier
      * @param authorizationServerEncodedUrl authorization server encoded URL
      */
-    public Builder(
+
+     public Builder(
+      AccessMethod method,
+      HttpTransport transport,
+      JsonFactory jsonFactory,
+      GenericUrl tokenServerUrl,
+      HttpExecuteInterceptor clientAuthentication,
+      String clientId,
+      String authorizationServerEncodedUrl) {
+    setMethod(method);
+    setTransport(transport);
+    setJsonFactory(jsonFactory);
+    setTokenServerUrl(tokenServerUrl);
+    setClientAuthentication(clientAuthentication);
+    setClientId(clientId);
+    setAuthorizationServerEncodedUrl(authorizationServerEncodedUrl);
+    setAdditionalParameters(Collections.<String, String>emptyMap());
```

Comment:
It is ok to just leave additional params as null. No need to set as empty map. It is ok to have it as well, no harm.